### DISCUSSION TITLE
Convert ical4j.model.Date to java.time.LocalDateTime

### DISCUSSION THREAD

**Discussion by rider87** [ical4j/ical4j#581] [General]
Hi all,

How can I convert the **net.fortuna.ical4j.model.Date** to **java.time.LocalDateTime** ?
Thank you for your help

Thanks

**Comment by benfortuna** [comment]
Hi, as the iCal4j `Date` and `DateTime` classes extend `java.util.Date`, you should be able to convert to an `Instant` first, and then `LocalDateTime`:

```java
net.fortuna.ical4j.model.Date mydate = ...
LocalDateTime ldt = LocalDateTime.from(mydate.toInstant());
```
